project-koku/koku
# Add Trino processing ENVs for source-type and accounts

**Testing**
1. Enable GCP for trino.  Ingest GCP and a non-GCP source and verify only GCP is processed with trino.

2. Enable acct10001 for trino.  Verify all sources are processed with trino for this account.


**Test Results**
```
1. Enable GCP for trino.  Ingest GCP and a non-GCP source and verify only GCP is processed with trino.
(koku2) ➜  koku2 git:(trino_new_envs) ✗ echo $ENABLE_TRINO_SOURCE_TYPE
GCP
(koku2) ➜  koku2 git:(trino_new_envs) ✗ 

GCP
koku-worker_1     | [2021-04-16 16:18:04,738] INFO f21ef62e-d490-4edf-ae9b-2dbd3279d31f {'message': 'Running convert_csv_to_parquet on file /testing/pvc_dir/processing/acct10001/gcp/2021-04-13.csv in S3 path data/csv/10001/GCP/source=0315d69b-5e1d-4285-9546-83195ef4b363/year=2021/month=04.', 'request_id': 'c610cb31-e769-4966-9e3c-22ebf9a24007', 'account': '10001', 'provider_uuid': '0315d69b-5e1d-4285-9546-83195ef4b363', 'provider_type': 'GCP'}


AWS:
koku-worker_1     | [2021-04-16 16:16:29,202] INFO f2f303a3-e597-4700-9de6-bdf033ddab21 File <_io.TextIOWrapper name='/testing/pvc_dir/processing/acct10001/aws/cost-usage-bucket/koku-00001.csv.gz' encoding='UTF-8'> opened for processing



2. Enable acct10001 for trino.  Verify all sources are processed with trino for this account.

(koku2) ➜  koku2 git:(trino_new_envs) ✗ echo $ENABLE_TRINO_ACCOUNTS
acct10001
(koku2) ➜  koku2 git:(trino_new_envs) ✗ echo $ENABLE_TRINO_SOURCE_TYPE

(koku2) ➜  koku2 git:(trino_new_envs) ✗ 


AWS
koku-worker_1     | [2021-04-16 17:08:00,361] INFO d533ada4-4830-406e-8b0d-722eed5a2900 {'message': 'Running convert_csv_to_parquet on file /testing/pvc_dir/processing/acct10001/aws/cost-usage-bucket/koku-00001.csv.gz in S3 path data/csv/10001/AWS/source=8a142ce7-50ef-4f2d-a2c5-d5255cf99020/year=2021/month=04.', 'request_id': '543b9004-f6ad-486b-96b1-517169433947', 'account': '10001', 'provider_uuid': '8a142ce7-50ef-4f2d-a2c5-d5255cf99020', 'provider_type': 'AWS'}


```
